src/main/java/com/example/bookstorewebapp/controller/BookController.java

```diff
@@ -0,0 +1,42 @@
+package com.example.bookstorewebapp.controller;
+
+import com.example.bookstorewebapp.dto.BookDto;
+import com.example.bookstorewebapp.dto.CreateBookRequestDto;
+import com.example.bookstorewebapp.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/books")
+@RequiredArgsConstructor
+public class BookController {
+    private static final String URL =
+            "https://peppy-horse-b3cd93.netlify.app/";
+    private static final String LOCAL_URL = "http://localhost:3000";
```

Comment:
Unfortunately, I even don't understand what is this). But I think that it's not so important on this level project.